src/pipecat/services/background_noise.py

```diff
@@ -0,0 +1,120 @@
+import asyncio
+import json
+import time
+from asyncio import sleep
+from io import BytesIO
+
+import loguru
+
+from pipecat.processors.frame_processor import FrameProcessor, FrameDirection
+from pydub import AudioSegment
```

Comment:
this is introducing a new dependency `pydub` we should add that  pyproject.toml and make sure we advise the use to do `pip install pipecat[pydub]`